packages/atlas/src/CommonProviders.tsx

```diff
@@ -34,20 +36,13 @@ export const CommonProviders: FC<PropsWithChildren> = ({ children }) => {
 const MaintenanceWrapper: FC<PropsWithChildren> = ({ children }) => {
   const { isKilled, wasKilledLastTime, error, loading } = useGetKillSwitch()
 
+  const shouldRenderMaintenance = isKilled || (error && wasKilledLastTime) || (loading && wasKilledLastTime)
+  const shouldRenderAdminModal = !loading && !error
```

Comment:
What's the reasoning behind this condition? What would be case in which we don't want to show the AdminModal?

Reply:
Sorry, you are right, it was pointless